README.md

```diff
@@ -22,18 +22,20 @@ This team is spun off from the [Modules team](https://github.com/nodejs/modules)
 
 ## Status
 
-### In Progress
+### Complete
 
 1. Finish https://github.com/nodejs/node/pull/37468 / https://github.com/nodejs/node/pull/35524, simplifying the hooks to `resolve`, `load` and `globalPreloadCode`.
 
-### Upcoming
-
 1. Refactor the internal Node ESM loader’s hooks into `resolve` and `load`. Node’s internal loader already has no-ops for `transformSource` and `getGlobalPreloadCode`, so all this really entails is merging the internal `getFormat` and `getSource` into one function `load`.
 
 1. Refactor Node’s internal ESM loader to move its exception on unknown file types from within `resolve` (on detection of unknown extensions) to within `load` (if the resolved extension has no defined translator).
```

Comment:
@JakobJingleheimer I think https://github.com/nodejs/node/pull/37468 did the first and third items, but not the second (refactor the internal Node ESM loader’s hooks); is this correct?

If so, maybe we should move that to the first “upcoming” item before chaining? At least for now. We can discuss whether it should be removed from the list, if you think perhaps it should no longer get done; but I feel like I _think_ we should still do it, as it feels right to have the internal hooks parallel the public ones.

Reply:
It did ½ did that item, and the other ½ we decided to not do at all because node's internal get_format is used directly (and merging it with get_source would break that). So instead, I created a new internal/default load wrapper that calls both of them.

Reply:
Someone's working on updating ts-node's loader to support the consolidated API, and we've hit the lack of `defaultGetFormat`; we're going to have to ship a copy of it.  Sharing as it relates to discussion in #26, which may affect future design work to expose this functionality via node's API, which may in turn affect internal refactoring decisions.

Reply:
So just to finish this PR, what should the “Refactor the internal Node ESM loader’s hooks” item say now, or should it be deleted from the list?

Reply:
@GeoffreyBooth updated 🙂